storage/storage_proto/src/lib.rs

```diff
@@ -371,6 +371,99 @@ impl IntoProto for GetStartupInfoResponse {
     }
 }
 
+/// Helper to construct and parse [`proto::storage::GetLatestLedgerInfosPerEpochRequest`]
+///
+/// It does so by implementing [`IntoProto`](#impl-IntoProto) and [`FromProto`](#impl-FromProto),
+/// providing [`into_proto`](IntoProto::into_proto) and [`from_proto`](FromProto::from_proto).
+#[derive(Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(any(test, feature = "testing"), derive(Arbitrary))]
+pub struct GetLatestLedgerInfosPerEpochRequest {
+    pub start_epoch: u64,
+}
+
+impl GetLatestLedgerInfosPerEpochRequest {
+    /// Constructor.
+    pub fn new(start_epoch: u64) -> Self {
+        Self { start_epoch }
+    }
+}
+
+impl FromProto for GetLatestLedgerInfosPerEpochRequest {
```

Comment:
We should be able to derive `FromProto`/`IntoProto`?

Reply:
I initially tried automated derivation but the compiler didn't like it, let me try again :)